src/set_theory/cardinal/cofinality.lean

```diff
@@ -1078,3 +1078,41 @@ begin
 end
 
 end cardinal
+
+section omega_1
+
+namespace ordinal
+open cardinal
+open_locale ordinal cardinal
+
+lemma sup_sequence_lt_omega_1 (o : ℕ → ordinal.{u}) (ho : ∀ n, o n < ω₁) :
+  sup o < ω₁ :=
+begin
+  apply sup_lt_ord_lift _ ho,
+  simp only [mk_denumerable, lift_aleph_0],
+  unfold omega_1,
+  rw [cardinal.is_regular_aleph_one.cof_eq],
+  exact aleph_0_lt_aleph_one,
+end
+
+/--
+Denumerably many elements chosen from a nondecreasing `ω₁`-sequence of sets,
+all lie in one of the sets.
+-/
+lemma bound_omega_1_of_increasing_of_sequence {β : Type*} {A : ordinal → set β}
```

Comment:
This is somewhat of a specialized lemma, I'd move it to the `borel_hierarchy` file.

Reply:
This lemma admits the same generalizations discussed above, and might be of use later, but for the time being I'm ok with the suggestion.